.github/workflows/go.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
-  pull_request_target:
```

Comment:
Removed this that it seems it's intermittent issue with uploading codecov report which doesn't depend on when running workflow on pull_request or pull_request_target: 

`{'detail': ErrorDetail(string='Repository slug or commit sha do not match Github actions build. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}`

It's solved by running workflow few time